lab-taylor-melanie-ken/lib/bitmap-writer.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+const constructor = require('./bitmap-constructor.js');
+const transform = require('./transform.js');
+
+const bitMapWriter = module.exports = function(file, newBM, transformMethod) {
+  let newPath = `${__dirname}/../data/${newBM}.bmp`;
+
+  fs.readFile(file, function(err,data) {
+    if(err) return (err);
+    let newBitMap = constructor(data);
+    console.log('transform method passed through:',transformMethod);
+
+    if (transformMethod === 'blkToWhite') {
```

Comment:
Is there a programmatic way you could handle this so you don't need so much repeated code? Take a look at Object.keys() for some inspiration.